### PR TITLE
Added "id" as an option for contextmenu item.

### PR DESCRIPTION
**Pull request by KJohnson74**
Added "id" as an option to be passed in per item in the menu object.

### CODE REVIEW

**Comment by nescalante** [comment]
the right file to change is [this one](https://github.com/nescalante/knockout.contextmenu/blob/master/src/knockout.contextmenu.js) (the one in the `src` folder) since the dist file is autogenerated by a `grunt` task

File: dist/js/knockout.contextmenu.js [hunk 1]
```diff
@@ -208,7 +212,7 @@ function bindContextMenu(ko, document) {
               classes.push('with-url');
             }
 
-            elements.push('<li class="' + classes.join(' ') + '">' + item.html + '</li>');
+            elements.push('<li id="' + id + '" class="' + classes.join(' ') + '">' + item.html + '</li>');
```
Comment (nescalante) [inline dist/js/knockout.contextmenu.js]: this should be `(id ? ('id="' + id '" ') : '')`

**Comment by KJohnson74** [comment]
Oh my bad. I'll change that and make another pull.

**Comment by nescalante** [comment]
LGTM

**Comment by KJohnson74** [comment]
I'm still fairly new to Git/GitHub so, I'm not quite sure how to rebase the commits properly yet. I'm working on figure it out now though.

**Comment by nescalante** [comment]
:+1: 
thanks for contributing!

**Pull request merged**